component/src/main/java/org/wso2/extension/siddhi/io/http/source/exception/HttpSourceAdaptorRuntimeException.java

```diff
@@ -18,18 +18,16 @@
  */
 package org.wso2.extension.siddhi.io.http.source.exception;
 
-import io.netty.handler.codec.http.HttpResponseStatus;
-import org.wso2.carbon.messaging.CarbonCallback;
-import org.wso2.carbon.messaging.CarbonMessage;
-import org.wso2.carbon.messaging.DefaultCarbonMessage;
-import org.wso2.carbon.transport.http.netty.common.Constants;
-
-import static io.netty.handler.codec.http.HttpVersion.HTTP_1_1;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.wso2.extension.siddhi.io.http.util.HttpIoUtil;
+import org.wso2.transport.http.netty.message.HTTPCarbonMessage;
 
 /**
  * HTTP source adaptor specific exception.
  */
 public class HttpSourceAdaptorRuntimeException extends RuntimeException {
+    private static final Logger log = LoggerFactory.getLogger(HttpSourceAdaptorRuntimeException.class);
```

Comment:
Where are we using this logger?

Reply:
Fixed in 156b93ab839dfc225e0a915f8951b61db7afa2ab